q2_types/distance_matrix/tests/test_type.py

```diff
@@ -0,0 +1,34 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, QIIME development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import unittest
+
+from q2_types.plugin_setup import plugin
+from q2_types.distance_matrix import (DistanceMatrix,
+                                      DistanceMatrixDirectoryFormat)
+from q2_types.testing import TestPluginBase
+
+
+class TestTypes(TestPluginBase):
+    def test_distance_matrix_semantic_type_registration(self):
+        self.assertEqual(plugin.types['DistanceMatrix'].semantic_type,
+                         DistanceMatrix)
```

Comment:
I disagree, this test is explicitly asserting the `DistanceMatrix` semantic type defined in this subpackage has been registered on the plugin object. Creating a helper method doesn't make it as clear what we're testing.